src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,475 @@
+
+[Maskable icons](https://web.dev/maskable-icon/) is a new icon format that makes icons for your PWA
+look great across all types of devices. To help your PWA look as good as possible, we've introduced
+a new audit to check if your manifest.json supports this new format.
+
+### Charset declaration {: #charset }
+
+The [meta charset element](https://web.dev/charset/) declares what character encoding should be used
+to interpret an HTML document. If this element is missing, or if it is declared late in the
+document, browsers employ a number of heuristics to guess which encoding should be used. If a
+browser guesses incorrectly, and a late meta charset element is found, the parser generally throws
+out all the work done so far and starts over, leading to poor experiences for the user. This new
+audit verifies the page has a valid character encoding and it's defined early and up front.
+
+## Lighthouse CI {: #ci }
+
+At [CDS last November](https://web.dev/lighthouse-evolution-cds-2019/#lighthouse-ci-alpha-release)
+we announced [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci), the open source Node
+CLI and server that tracks Lighthouse results on every commit in your continuous integration
+pipeline, and we've come a long way since the alpha release. Lighthouse CI now has native support
+for numerous CI providers including Travis, Circle, GitLab, and Github Actions. Ready-to-deploy
+[docker images](https://github.com/GoogleChrome/lighthouse-ci/tree/master/docs/recipes) make setup a
+breeze, and a comprehensive dashboard redesign now reveals trends across every category and metric
+in Lighthouse for detailed analysis.
+
+Start using Lighthouse CI on your project today by following our
+[getting started guide](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/getting-started.md).
+
+<style>
+@media (min-width: 865px) {
+  .w-figure--inline-left {
+    margin: 0 20px 40px 0;
+  }
+}
+</style>
+
+<figure class="w-figure w-figure--inline-left">
+  <a href="lighthouse-ci-1.png" target="_blank">
+    <img class="w-screenshot" src="lighthouse-ci-1.png" alt="Lighthouse CI." width="600">
+  </a>
+</figure>
+<figure class="w-figure w-figure--inline-left">
+  <a href="lighthouse-ci-2.png" target="_blank">
+    <img class="w-screenshot" src="lighthouse-ci-2.png" alt="Lighthouse CI." width="600">
+  </a>
+</figure>
+<figure class="w-figure">
+  <a href="lighthouse-ci-3.png" target="_blank">
+    <img class="w-screenshot" src="lighthouse-ci-3.png" alt="Lighthouse CI." width="600">
+  </a>
+</figure>
+
+## Renamed Chrome DevTools panel {: #devtools }
+
+We've renamed the **Audits** panel to the **Lighthouse** panel. Enough said!
+
+Depending on your DevTools window size, the panel is probably behind the `»` button. You can drag
+the tab to change the order.
+
+To quickly reveal the panel with the [Command
+menu](https://developers.google.com/web/tools/chrome-devtools/command-menu): 
+
+1. {% Instruction 'devtools' %}
```

Comment:
don't know why but the formatting here got odd

![image](https://user-images.githubusercontent.com/39191/82251822-c7d79480-9902-11ea-97a3-0c56f2deb114.png)

Reply:
My bad, I know what it is 🤦🏻‍♀️ I've suggested a fix 👍